cfgov/retirement_api/jinja2/retirement_api/claiming.html

```diff
@@ -38,6 +38,23 @@ <h1>{{ _("Planning your Social Security claiming age") }}</h1>
                     <a href="es/">Ver página en español</a>
                     {% endif %}
                 </p>
+                {% if current_language == 'en' %}
+                <div class="o-well">
+                    <p>The Social Security Administration offers
+                         <a class="a-link a-link__icon" href="https://www.ssa.gov/multilanguage/interpreter.html">free interpreter services</a>
+                         and a <a class="a-link a-link__icon" href="https://www.ssa.gov/site/languages/en/">multilanguage gateway</a>
+                         to help consumers who speak other languages.
+                    </p>
+                </div>
+                {% else %}
+                <div class="o-well">
+                    <p>La Administración del Seguro Social ofrece
+                         <a class="a-link a-link__icon" href="https://www.ssa.gov/multilanguage/interpreter.html">servicios de intérprete</a>
+                         y un <a class="a-link a-link__icon" href="https://www.ssa.gov/site/languages/en/">portal multilingüe</a>
+                         para ayudar a los consumidores que hablan otros idiomas.
+                    </p>
+                </div>
+                {% endif %}
```

Comment:
The `o-well` could probably go outside the conditional.

```suggestion
                <div class="o-well">
                    {% if current_language == 'en' %}
                    <p>The Social Security Administration offers
                         <a class="a-link a-link__icon" href="https://www.ssa.gov/multilanguage/interpreter.html">free interpreter services</a>
                         and a <a class="a-link a-link__icon" href="https://www.ssa.gov/site/languages/en/">multilanguage gateway</a>
                         to help consumers who speak other languages.
                    </p>
                    {% else %}
                    <p>La Administración del Seguro Social ofrece
                         <a class="a-link a-link__icon" href="https://www.ssa.gov/multilanguage/interpreter.html">servicios de intérprete</a>
                         y un <a class="a-link a-link__icon" href="https://www.ssa.gov/site/languages/en/">portal multilingüe</a>
                         para ayudar a los consumidores que hablan otros idiomas.
                    </p>
                    {% endif %}
                </div>
```

Maybe the `p` too? But for some reason I'm less sure about it.